vmdb/spec/models/ems_refresh/refreshers/openshift_refresher_spec.rb

```diff
@@ -104,8 +104,7 @@ def assert_specific_container_route
                                 :ems_ref      => "ff8a8e45-e385-11e4-9d96-f8b156af4ae1",
                                 :name         => "route-edge",
                                 :namespace    => "default",
-                                :host_name    => "www.example.com",
-                                :service_name => "frontend"
+                                :host_name    => "www.example.com"
                             )
```

Comment:
Can you add something like:

@container_route.container_service.name.should == "frontend"